tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -248,12 +249,13 @@ void onRenderTaskPostExecute() {
     if( mTileRenderListener != null ) {
       mTileRenderListener.onRenderComplete();
     }
-    invalidate();
-    requestRender();
```

Comment:
This is in case there was any changes to the tiles to be displayed since the render process started.    If there is no change, it'll fast-fail out anyway.